deployments/continuous-deployment-config/ocis_full/production.yml

```
@@ -36,9 +36,9 @@
           COMPANION_IMAGE: owncloud/uppy-companion:3.12.13-owncloud
           WOPISERVER_DOMAIN: wopiserver.ocis.production.owncloud.works
           COLLABORA_DOMAIN: collabora.ocis.production.owncloud.works
-          INBUCKET_DOMAIN: mail.ocis.production.owncloud.works
+          MAIL_SERVER_DOMAIN: mail.ocis.production.owncloud.works
```

Just a suggestion for consistency with other service env variable names, but not mandatory.
```suggestion
          MAIL_DOMAIN: mail.ocis.production.owncloud.works
```

---

I think it will not create confusions